src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -238,4 +240,12 @@ void testGetPredefinedShelfNameReturnsCorrectShelf(String shelfName) {
                 predefinedShelfUtils.getPredefinedShelfName(shelfName);
         assertEquals(expectedShelf, actualShelf);
     }
+    
+    @Test
+    void testGetPredefinedShelfNamesAsStrings() {
+    	List<String> actualShelfNames = predefinedShelfUtils.getPredefinedShelfNamesAsStrings();
+    	List<String> expectedShelfNames = Stream.of(ShelfName.values()).map(Enum::toString).collect(Collectors.toList());
```

Comment:
Could you bring the initialisation down one line so that it adheres to the 100 character column limit specified in our style guide?

Reply:
Thank you for helping me formatted these lines.